src/components/UserList/UserListToolbar.tsx

```diff
@@ -46,8 +44,8 @@ export function UserListToolbar(props: UserListToolbarProps) {
           ) : (
             <></>
           )}
-          {selectedRows.every((row) => {
-            return row.userType !== 'Admin';
+          {selected.every((row) => {
+            return row.rolesId !== '1';
```

Comment:
this should be an enum. You can probably get this type from generated open API types